sapl/painel/views.py

```diff
@@ -358,37 +361,57 @@ def get_dados_painel(request, pk):
     elif expediente:
         return JsonResponse(get_presentes_expediente(pk, response, expediente))
 
-    ultima_ordem = get_last_materia(pk)
-
-    if ultima_ordem:
-        if ultima_ordem.resultado:
-            if ultima_ordem.tipo_votacao in [1, 3]:
-                return JsonResponse(
-                    get_votos(get_presentes(
-                        pk, response, ultima_ordem), ultima_ordem))
-            elif ultima_ordem.tipo_votacao == 2:
-                return JsonResponse(
-                    get_votos_nominal(get_presentes(
-                        pk, response, ultima_ordem), ultima_ordem))
-        else:
-            return JsonResponse(get_presentes(pk, response, ultima_ordem))
-
-    ultimo_expediente = get_last_materia_expediente(pk)
-
-    if ultimo_expediente:
-        if ultimo_expediente.resultado:
-            if ultimo_expediente.tipo_votacao in [1, 3]:
-                return JsonResponse(
-                    get_votos(get_presentes(
-                              pk, response, ultimo_expediente),
-                              ultimo_expediente))
-            elif ultimo_expediente.tipo_votacao == 2:
-                return JsonResponse(
-                    get_votos_nominal(get_presentes(
-                                      pk, response, ultimo_expediente),
-                                      ultimo_expediente))
-        else:
+    # Ultimo voto em ordem e ultimo voto em expediente
+    last_ordem_voto = RegistroVotacao.objects.filter(
+        ordem__sessao_plenaria=sessao).last()
+    last_expediente_voto = RegistroVotacao.objects.filter(
+        expediente__sessao_plenaria=sessao).last()
+
+    # Ultimas materias votadas
+    if last_ordem_voto:
+        ultima_ordem_votada = last_ordem_voto.ordem
+    if last_expediente_voto:
+        ultimo_expediente_votado = last_expediente_voto.expediente
+
+    # Caso não tenha nenhuma votação aberta
+    if last_ordem_voto or last_expediente_voto:
+
+        # Se alguma ordem E algum expediente já tiver sido votado...
+        if last_ordem_voto and last_expediente_voto:
+            # Verifica se o último resultado é um uma ordem do dia
+            if last_ordem_voto.pk >= last_expediente_voto.pk:
+                if ultima_ordem_votada.tipo_votacao in [1, 3]:
```

Comment:
Coloca assim:

COD_NOMINAL = 2
COD_SECRETA = 1
COD_SIMBOLICA = 3

if ultima_ordem_votada.tipo_votacao in [COD_NOMINAL, COD_SECRETA]: